### ISSUE TITLE
Context menu is partly visible when properties panel is expanded widely

### ISSUE THREAD

**Issue by ThorbenLindhauer** [camunda/camunda-modeler#2024]
__Describe the Bug__

With little canvas space, the context menu is hidden so that not all entries can be accessed. This is expected to some extent, but can maybe be improved in some scenarios. I discussed this with @philippfromme on slack.

__Steps to Reproduce__

1. Expand the properties panel fairly widely (e.g. 50% of screen)
2. Click wrench button on boundary event so that there is not enough canvas space for the menu to appear on the right side of the boundary event
3. Context menu is displayed to the top left side of the boundary event, most entries are hidden (see screenshot)

![grafik](https://user-images.githubusercontent.com/586648/100465613-e7c8eb80-30cf-11eb-9ea6-b80838308a8b.png)

__Expected Behavior__

It looks like we can make better use of the available space to display as much of the context menu as possible.

__Environment__

- OS: Windows 10
- Camunda Modeler Version: 4.4.0
- Resolution: 1920x1080 (150% text size)

**Comment by philippfromme** [comment]
Excellent, thanks for reporting. 👍

**Comment by nikku** [comment]
To better understand this issue: Given that small amount of available screen, what would be your expectations? 

* That the context menu is shown in the middle of the Canvas
* That the context menu is shown on top of Canvas _and_ app UI (i.e. above tabs)
* Both of these?
* Any other option?

**Comment by ThorbenLindhauer** [comment]
Both of the options sound reasonable to me. It is ok if this degrades the experience a little (by hiding other elements that I might want to click next), as I click deliberately on the wrench with the goal to select one of the options. As long as I can easily collapse the menu again, that's fine for me (i.e. there should be some space left on the canvas that I can click to make it go away).

**Comment by barmac** [comment]
This issue is fixed now:

![image](https://github.com/camunda/camunda-modeler/assets/28307541/55e7fa74-d984-499d-90c8-2c89f98d9d30)
